edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubConfigParser.cs

```diff
@@ -52,14 +62,63 @@ public static EdgeHubConfig GetEdgeHubConfig(EdgeHubDesiredProperties desiredPro
             return new ReadOnlyDictionary<string, RouteConfig>(routes);
         }
 
-        static Option<BrokerConfig> ParseBrokerConfig(BrokerProperties configuration)
+        Option<BrokerConfig> ParseBrokerConfig(BrokerProperties properties)
         {
-            if (configuration != null)
+            if (properties != null)
             {
-                throw new NotImplementedException();
+                return Option.Some(
+                    new BrokerConfig(
+                        this.ParseBridgeConfig(properties),
+                        this.ParseAuthorizationConfig(properties)));
             }
 
             return Option.None<BrokerConfig>();
         }
+
+        /// <summary>
+        /// EH Twin and policy definition in the Broker have different json schemas.
```

Comment:
Why is it that twin and policy definition have different schemas?

Reply:
This is what Varun and Emanuel have decided.